cli/src/commands/branch/create.ts

```diff
@@ -31,12 +31,17 @@ export default class BranchCreate extends BaseCommand<typeof BranchCreate> {
 
     const { from } = flags;
 
-    const result = await xata.api.branches.createBranch({ workspace, region, database, branch, from });
-
-    if (this.jsonEnabled()) return result;
-
-    const message = `Branch ${branch} successfully created`;
-
-    this.success(message);
+    try {
+      const result = await xata.api.branches.createBranch({ workspace, region, database, branch, from });
+
+      if (this.jsonEnabled()) return result;
+
+      const message = `Branch ${branch} successfully created`;
+      this.success(message);
+    } catch (err) {
+      err instanceof Error
+        ? this.error(`${err.message.includes('Unexpected token < in JSON') ? 'Failed to create branch' : err.message}`)
```

Comment:
Checking for the string that we know if thrown during a timeout. Could alternatively override all error messages from backend but I figured that some of them may actually be useful.